src/Ilios/CoreBundle/Controller/OfferingController.php

```diff
@@ -237,6 +247,27 @@ public function putAction(Request $request, $id)
                 $code = Codes::HTTP_CREATED;
             }
 
+            // capture the values of offering properties pre-update
```

Comment:
ok. will hide the ugliness of this in a utility function. offering entity is as good/bad as any of the other places to do so.